Explain variable escaping (#25410)
fleetdm/fleet: articles/secret-variables.md

_Note:_ The checksum of Apple DDM profiles with secret variables now includes the timestamp of the last secrets update.

## Escaping variable and interpolation

> The dollar sign ($) can be escaped so it's not considered a variable by using a backslash (e.g. `\$100`). Additionally, `${variable}` syntax can be used (e.g. if you want to use a variable as part of the string `"Bearer ${FLEET_SECRET_SOME_API_TOKEN}"`).

Reviewer: ```suggestion
The dollar sign ($) can be escaped so it's not considered a variable by using a backslash (e.g. `\$100`). Additionally, `MY${variable}HERE` syntax can be used if you want to put strings around the variable.
```

Reviewer: Thanks @getvictor! I updated this section.

FYI @marko-lisica

> The dollar sign ($) can be escaped so it's not considered a variable by using a backslash. (e.g. `\$100`). Additionally, `${variable}` syntax can be used (e.g. if you want to use a variable as part of the string `"Bearer ${FLEET_SECRET_SOME_API_TOKEN}"`).

Reviewer: ```suggestion
The dollar sign ($) can be escaped so it's not considered a variable by using a backslash (e.g. `\$100`). Additionally, `${variable}` syntax can be used (e.g. if you want to use a variable as part of the string `"Bearer ${FLEET_SECRET_SOME_API_TOKEN}"`).
```

Removing double period.
